### PR TITLE
Pres hotfixes

### PR DESCRIPTION
**Pull request by WoozyDragon**
Only merge after the presentation is done

### CODE REVIEW

File: lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/robot.cpp [hunk 1]
```diff
@@ -91,7 +91,9 @@ void cb(int8_t speed) { exc_mtr.write(speed); }
 } // namespace excavation
 
 namespace deposition {
-Sabertooth_MotorCtrl dep_mtr{&MC3, STMotor::M1};
+Sabertooth_MotorCtrl dep_mtr{&MC3, STMotor::M2};
+//Sabertooth_MotorCtrl exc_mtr{&MC3, STMotor::M1};
+
```
Comment (raghavauppuluri13) [inline lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/robot.cpp:96]: Is excavation not working right now? any updates to this?

File: lunabot_control/scripts/manual_controller.py [hunk 2]
```diff
@@ -133,9 +134,19 @@ def joy_callback(self, joy):
                 effort_msg.excavate = self._latched_excavation_speed
             else:
                 # Change the range of the trigger axis from [1, -1] to [0, 1]
-                right_trigger_axis_normalized = (-joy.axes[Axes.RIGHT_TRIGGER.value] + 1) / 2
-                left_trigger_axis_normalized = (-joy.axes[Axes.LEFT_TRIGGER.value] + 1) / 2
+                if (joy.axes[Axes.RIGHT_TRIGGER.value] == 0):
+                    right_trigger_axis_normalized = 0
+                    #print("normR")
```
Comment (ashviniyer21) [inline lunabot_control/scripts/manual_controller.py:139]: Delete print statements?

**Review by raghavauppuluri13 — APPROVED** [review]

File: lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/robot.cpp [hunk 3]
```diff
@@ -25,7 +25,7 @@ void update(int32_t &act_right_curr) {
 }
 
 void cb(int8_t lin_act_volt) {
-  act_left_mtr.write(-lin_act_volt);
+  act_left_mtr.write(lin_act_volt);
```
Comment (ashviniyer21) [inline lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/robot.cpp:28]: Future note, can we have a common place to store inversions? Imo we should assume right directions in software, have some booleans in firmware to store if values are inverted or not so its easy to tell / fix if wiring ever changes

File: lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/robot.cpp [hunk 4]
```diff
@@ -92,6 +92,8 @@ void cb(int8_t speed) { exc_mtr.write(speed); }
 
 namespace deposition {
 Sabertooth_MotorCtrl dep_mtr{&MC3, STMotor::M1};
+//Sabertooth_MotorCtrl exc_mtr{&MC3, STMotor::M1};
```
Comment (ashviniyer21) [inline lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/robot.cpp:95]: If we know this works delete commented code?

**Review by ashviniyer21 — APPROVED** [review]
Left minor comments, otherwise LGTM

File: lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/robot.cpp (hunk 3 above)
Comment (raghavauppuluri13) [reply lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/robot.cpp:28]: this is easily readable tbh. having hardware level hotfixes in the robot.cpp file is pretty minimal solution imp. the ROS topic api is assumed to be standardized and metric units.

**Pull request merged**